### PR TITLE
Dns dock version

### PR DESCRIPTION
**Pull request by jngermon**
With docker-compose write in version 2 and dnsdock:latest, the system is not working. Dnsdock:latest do not recognonize container IP. We have to use the "amd64-1.13.1" version (5 months younger) to fix it.

### CODE REVIEW

File: bin/docker-outline-dns [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Get docker ip
+docker_ip=$(ip -family inet -oneline addr show docker0|\
```
Comment (greg0ire) [inline bin/docker-outline-dns:4]: Is it outline or oneline? Pick one ;)

File: bin/docker-outline-dns [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    cut --delimiter=\  --fields=7|cut --delimiter=/ --fields=1)
+
+# Add dns as nameserver
+resolvconf -u
+sed --in-place "1i options timeout:1\nnameserver ${docker_ip}" /run/resolvconf/resolv.conf
+
```
Comment (greg0ire) [inline bin/docker-outline-dns]: Please remove line 10.

File: bin/docker-outline-dns [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Get docker ip
+docker_ip=$(ip -family inet -oneline addr show docker0|\
+    cut --delimiter=\  --fields=7|cut --delimiter=/ --fields=1)
```
Comment (greg0ire) [inline bin/docker-outline-dns]: `--delimiter=' '` is maybe easier to understand

**Review by greg0ire — CHANGES_REQUESTED** [review]

File: bin/docker-outline-dns (hunk 1 above)
Comment (jngermon) [reply bin/docker-outline-dns:4]: it's "oneline" option for "ip" command"  => output one record per line

File: bin/docker-outline-dns (hunk 3 above)
Comment (jngermon) [reply bin/docker-outline-dns]: Ok I will do ti

File: bin/docker-outline-dns (hunk 1 above)
Comment (greg0ire) [reply bin/docker-outline-dns:4]: Ok, but the file is named docker-outline-dns… what does that refer to?

Comment (greg0ire) [reply bin/docker-outline-dns:4]: ping @jngermon

Comment (jngermon) [reply bin/docker-outline-dns:4]: Apology
The two files are about two differents things.
For the dns_dock version, only the change in docker-init is requiered.
The file docker-outline-dns is to enable use this system on outline mode (without connection the network manager does not link DNSDOCK as DNS server.
Do you want two differents Pull requests ? (or just one about docker-compose v2 ?)

Comment (greg0ire) [reply bin/docker-outline-dns:4]: Ooooh ok I think you mean "offline mode", don't you? Can you change that? Also, one PR is fine, as long as you make 2 commits :)

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Totally forgot about this PR, thanks @jngermon !

**Pull request merged**